src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -513,7 +513,7 @@ private void ProcessPayload(FormatEntryData fed, FormatMessagesContextManager.Ou
 
             if (fed.formatEntryInfo == null)
             {
-                PSTraceSource.NewArgumentNullException("fed.formatEntryInfo");
```

Comment:
I wonder why do we change this? Please revert.

Reply:
We should not use `ArgumentNullException` here as the argument is not null but a property.

Reply:
Intention is _to trace_. We shouldn't remove this.

Reply:
I wonder why we do not throw the exception from `NewArgumentNullException` here. I think this could be a bug.